src/rviz/CMakeLists.txt

```
@@ -1,5 +1,7 @@
 find_package(yaml-cpp REQUIRED)
 
+include_directories(${YAML_CPP_INCLUDE_DIR})
```

Please mark them as system includes:
```suggestion
include_directories(SYSTEM ${YAML_CPP_INCLUDE_DIR})
```